src/contact.js

```diff
@@ -0,0 +1,47 @@
+export function loadContactPage() {
```

Comment:
To much code repeated, try to break this function using some helper functions to avoid repetation.